libr/bin/format/elf/elf_specs.h

```diff
@@ -159,6 +159,7 @@
 #define EM_56800EX             200        /* Freescale 56800EX Digital Signal Controller (DSC) */
 #define EM_BA1                 201        /* Beyond BA1 CPU architecture */
 #define EM_BA2                 202        /* Beyond BA2 CPU architecture */
+#define EM_BA2_NON_STANDARD 0x8472        /* Beyond BA2 CPU architecture */
```

Comment:
I would suggest to add in the comment a link to NXP doc that defines this.

Reply:
Actually I extracted this value from elf files generated by the NXP toolchain.
I'll check if I find a documentation somewhere....

Reply:
Ah. ok, if this is not mapped anywhere, let me know, and i'll merge this.

Reply:
I found a reference in the binutils version 2.22:
`binutils-2.22-ba-r33675/include/elf/common.h:362:#define EM_BA			0x8472`
A tarball of the corresponding binutils can be found here: https://github.com/alephsecurity/BA2-toolchain

Reply:
hmm. then don't worry.